Client/WebFilters/FilterList.swift

```diff
@@ -11,6 +11,9 @@ struct FilterList: Decodable, Identifiable {
     case uuid, title, componentId, description = "desc", urlString = "url"
   }
   
+  public static let mobileAnnoyancesComponentID = "ldbgldhcahahpffloggbbmjllggnkenk"
+  public static let mobileAnnoyancesUUID = "2F3DCE16-A19A-493C-A88F-2E110FBD37D6"
```

Comment:
This is a good approach but I decided on a slightly different one. Perhaps we can put all the static uuids in one place. Mine are in `FilterListResourceDownloader` starting on line 118 here:

```
/// The uuid of the cookie consent notices filter list.
  ///
  /// This is a special filter list that has more accessible UI to control it
  public static let cookieConsentNoticesUUID = "AC023D22-AE88-4060-A978-4FEEEC4221693"
  /// The uuid of the "Remove Web annoyances" filter list
  ///
  /// This is a special filter list that has more accessible UI to control it
  public static let removeWebAnnoyancesUUID = "67E792D4-AE03-4D1A-9EDE-80E01C81F9B8"
```

If you prefer your approach maybe you can move my UUIDs to `FilterList`

Also I thought we were going to enable the web annoyances filter list so I added the `removeWebAnnoyancesUUID` but if we're not going to use it perhaps we should remove that one too. I can do that but if you will touch the code perhaps you don't mind doing it

Reply:
As discussed this code will be available after this PR so its me that needs to sync up in my PR with this one. So Ignore this comment.